### DISCUSSION TITLE
2.1.1rc3

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#25019] [General]
The release candidate:
https://dist.apache.org/repos/dist/dev/superset/2.1.1rc3/

Git tag for the release:
https://github.com/apache/superset/tree/2.1.1rc3

The Change Log for the release:
https://github.com/apache/superset/blob/2.1.1rc3/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/2.1.1rc3/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

**Full Changelog**: https://github.com/apache/superset/compare/2.1.0...2.1.1rc3

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/2.1.1rc3'>2.1.1rc3</a>.</em>

**Comment by Cmagno13** [comment]
Good morning!
Doesn't this version have new graphics?

**Reply by eschutho** [reply]
This was answered in slack, but I'll paste @rusackas' answer here for anyone viewing this in the discussion:
>Minor releases are generally bugfix/security releases. You can check the changelog for things relevant to charts there. New charts would be in minor or major releases, though I don't think any new charts have been contributed. If you'd like to contribute one you feel is missing, or improve one that's not meeting your needs, that'd be great!
